processing/thermal.py

```diff
@@ -60,18 +60,51 @@ def process(recording, conf):
             classify(conf, recording, api, s3, logger)
 
 
-def classify(conf, recording, api, s3, logger):
-    working_dir = recording["filename"].parent
-    command = conf.classify_cmd.format(
-        folder=str(working_dir), source=recording["filename"].name
-    )
+def classify_models(api, command, conf):
+    """ classifies all models described in the config """
 
-    logger.debug("processing %s", recording["filename"])
+    model_results = []
+    main_model = None
+
+    for model in conf.models:
+        model_result = classify_file(api, command, conf, model=model)
+        if model.live:
+            main_model = model_result
+
+        model_results.append(model_result)
+
+    return model_results, main_model
+
+
+def classify_file(api, command, conf, model=None):
+    if model and model.model_file:
+        command = "{} -m {}".format(command, model.model_file)
+
+    classify_info = run_classify_command(command, conf.classify_dir)
+
+    track_info = classify_info["tracks"]
+    formatted_tracks = format_track_data(track_info)
+
+    # Auto tag the video
+    tagged_tracks, tags = calculate_tags(formatted_tracks, conf)
 
+    model_result = {
+        "tracks": tagged_tracks,
+        "tags": tags,
+        "algiorithm_id": api.get_algorithm_id(classify_info["algorithm"]),
+    }
+    if model:
+        model_result["live"] = model.live
```

Comment:
Again what if none are marked live? Also 'live' is probably not the best description.  'Current', 'track creater' maybe because both models are live.   What exactly do you mean by live?